### PR TITLE
Bump swiftcord cask to v0.5.1

### PR DESCRIPTION
**Pull request by cryptoAlgorithm**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

**Comment by cryptoAlgorithm** [comment]
My bad, forgot to run the style checks before committing

File: Casks/swiftcord.rb [hunk 1]
```diff
@@ -22,4 +22,8 @@
     "~/Library/Containers/io.cryptoalgo.swiftcord",
     "~/Library/Saved Application State/io.cryptoalgo.swiftcord.savedState",
   ]
+
+  caveats "If you're updating from an older version of Swiftcord, you might get stuck on the loading screen " \
+          "due to a bug in the new account storage system. Clicking the logout button that appears after ~30s " \
+          "is a workaround which will allow you to log in again. You'll only have to do this once."
```
Comment (SMillerDev) [inline Casks/swiftcord.rb:28]: Is this specific to Homebrew?

Comment (cryptoAlgorithm) [reply Casks/swiftcord.rb:28]: Nope, but it's very important that users see that, especially when running brew upgrade where users might not check the project's readme

Comment (SMillerDev) [reply Casks/swiftcord.rb:28]: Caveats are really only for Homebrew issues.

Comment (cryptoAlgorithm) [reply Casks/swiftcord.rb:28]: Ok, I'll remove this stanza then. Is there a stanza more suitable for this purpose?

Comment (SMillerDev) [reply Casks/swiftcord.rb]: There is not, and since people generally do not read their package manager output I doubt it would help.

**Review by p-linnane — APPROVED** [review]

**Pull request closed without merging**